bsc.bugs/b381/b381.exp

```diff
@@ -1,5 +1,9 @@
-# bug 381 is not fixed, but an unknown-size error masks it
-compile_fail_error Converter.bs T0035
+# bug 381 is not fixed, we should raise an error here
+compile_fail_bug Converter.bs T0035
+
+# We now get an internal error when Converter is synthesized in another module
+compile_verilog_fail YY.bsv
+compare_file_filtered YY.bsv.bsc-vcomp-out {} "s/Bluespec Compiler \(build .*\)/d"
```

Comment:
The output of the `Bluespec Compiler` line varies depending on whether the source has version info defined in it.  Right now, the version number is empty, so only the build tag is displayed.  This filter line should probably just look for `Bluespec Compiler` at the start of the line, and not rely on matching what follows.

Reply:
Done.